docs/documentation/updating-the-kit.md

```diff
@@ -23,9 +23,9 @@ Tell us as much as you can about the issue you're having, and the computer and o
 In Finder on Mac or Windows Explorer, go to your prototype folder and open the file `VERSION.txt`. This will show what version of the Prototype Kit you’re using.
 
 - If your prototype is version 8 or before, then contact the GOV.UK Prototype team for help with updating it
-- If your prototype is version 9, 10 or 11, you can update to version 12 by following the steps on this page
+- If your prototype is version 9, 10, 11, 12.0.1 or 12.0.2, you can update to version 12.0.3 by following the steps on this page
```

Comment:
wondering if this might work? Avoids users having to check a list which could get longer, and us having to keep it up to date

> If your prototype is version 9, 10, 11, or 12 you can update to the latest version by following the steps on this page